src/_angular/font-setting/dtv-font-setting.js

```diff
@@ -383,6 +410,51 @@
               sheet.addRule("@font-face", rule);
             }
           }
+
+          function addVerticalAlignButtons(editor) {
+            editor.addButton("aligntop", {
+              image: "//s3.amazonaws.com/Rise-Images/Icons/align-top.svg",
```

Comment:
@donnapep This is the first angular directive from this repo that requires an image. In this situation we're vulnerable to an incorrect path of where the image needs to reside and it is easy to forget to include a gulp task in a widget repo to move the image where it needs to be. Personal preference is to just host the image in the bucket.